src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--5 div {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 div:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 div:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 div:nth-child(-n + 3) {
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead of selecting `div:nth-child(-n + 3)` directly within `.stars--3`, use a class that you assign to the children of `.stars--3`.